src/dynamic_importer/main.py

```diff
@@ -13,6 +14,26 @@
 from dynamic_importer.util import validate_env_values
 
 CREATE_DATA_MSG_INTERVAL = 10
+DIRS_TO_IGNORE = [
```

Comment:
Should we think about a `--exclude-dirs` option to allow users to extend this list?